### ISSUE TITLE
PyTorch DataLoaders

### ISSUE THREAD

**Issue by wjmaddox** [OATML/bdl-benchmarks#3]
Hi,

I'm looking forward to contributing a couple of our benchmarks into this repo, but am not seeing PyTorch dataloaders (or really any PyTorch support). Are there still plans to have PyTorch data loader support for the segmentation tasks?

Additionally, is there a timeline for making public the other benchmarks in the `pre-alpha` branch?

Thanks,
Wesley

**Comment by filangelos** [comment]
Hi @wjmaddox,

`NumPy` data loaders are already [provided](https://github.com/OATML/bdl-benchmarks/blob/alpha/bdlb/diabetic_retinopathy_diagnosis/benchmark.py#L211), which are framework-agnostic. Take a look at this [`colab`](https://colab.research.google.com/drive/1wSzFIEbzf2wuERrsad1JlTVk9XPCSCuF), where a simple `PyTorch` model is trained using them!

As far as the other benchmarks go:
* `Autonomous Vehicle's Scene Segmentation` will be **soon** made public,
* `Galaxy Zoo` and `Fishyscapes` are owned by collaborators and due to this dependency, we cannot really guarantee anything.

**Comment by wjmaddox** [comment]
Gotcha, thanks for the response! I'll check out that Colab notebook. 

Additionally, is the suggested mechanism for using data augmentation in Pytorch to 1) use the numpy dataloaders 2) convert to pytorch and 3) perform data augmentation using torchvision on the pytorch tensors?

**Comment by PolinaKirichenko** [comment]
Hi @filangel,

Thanks for the library! I was also wondering if you planned to implement / open source pytorch support? It would be very helpful to have it for things like data augmentation / data shuffling as noted by @wjmaddox. Or would you have any suggestions on what would be the best way to wrap it in pytorch dataloader?

Thank you!
